dir.yaml

```
@@ -470,6 +469,12 @@
     - title_en: Command Line Interface
       title_cn: 命令行接口
       path: admin/cli
+    - title_en: Durable MQTT Sessions
```

in the durability/durability_introduction file, the title is MQTT Durable Sessions
```suggestion
    - title_en: MQTT Durable Sessions
```